### PR TITLE
[12.0][MIG] website_lazy_load_image

### PR DESCRIPTION
**Pull request by oscarolar**

### CODE REVIEW

File: website_lazy_load_image/templates/snippets.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<template id="snippet_options" inherit_id="website.snippets">
```
Comment (tbaden) [inline website_lazy_load_image/templates/snippets.xml:7]: optional change: 
use original id (snippets)

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM

**Comment by simahawk** [comment]
Hola hombre :)
Issues worth to consider while migrating: https://github.com/OCA/website/issues/551 https://github.com/OCA/website/pull/571

**Comment by hugho-ad** [comment]
@simahawk 

This work will continue at https://github.com/OCA/website/pull/656

**Comment by hugho-ad** [comment]
@tbaden 

FYI this will continue at https://github.com/OCA/website/pull/656, so please could you help us with a new review

**Pull request closed without merging**